### PR TITLE
docs(APIv2): RHINENG-14358 spec documentation

### PR DESCRIPTION
**Pull request by romanblanco**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: spec/integration/v2/rules_spec.rb [hunk 1]
```diff
@@ -355,7 +356,7 @@
 
       v2_auth_header
       tags 'Policies'
-      description 'Unassigns a Rule from a Tailoring'
+      description 'Customizing of the tailoring by removing rule'
```
Comment (CodeDuchess) [inline spec/integration/v2/rules_spec.rb:359]: I think is a little confusing. Change to:
Use this to remove a rule from your tailoring.

File: swagger/v2/openapi.json [hunk 2]
```diff
@@ -101,7 +101,7 @@
         "tags": [
           "Policies"
         ],
-        "description": "Lists Policies",
+        "description": "You can see what policies are currently in place.Policies are statements that manage API scenariossuch as authentication, rate limiting, caching, requests and responses.",
```
Comment (CodeDuchess) [inline swagger/v2/openapi.json:104]: need a space between scenarios and such.

File: swagger/v2/openapi.json [hunk 3]
```diff
@@ -853,7 +853,7 @@
         "tags": [
           "Systems"
         ],
-        "description": "Lists Policies under a System",
+        "description": "List all policies assigned to a single system",
```
Comment (CodeDuchess) [inline swagger/v2/openapi.json:856]: Missing a period at the end.

File: swagger/v2/openapi.json [hunk 4]
```diff
@@ -6508,7 +6508,7 @@
         "tags": [
           "Policies"
         ],
-        "description": "Unassigns a Rule from a Tailoring",
+        "description": "Customizing of the tailoring by removing rule",
```
Comment (CodeDuchess) [inline swagger/v2/openapi.json:6511]: change to: Use this endpoint to remove a rule from your tailoring profile.

**Review by CodeDuchess — COMMENTED** [review]
@romanblanco Hey Roman. I have a few suggestions. Otherwise it looks great!

File: spec/integration/v2/policies_spec.rb [hunk 5]
```diff
@@ -14,7 +14,9 @@
     get 'Request Policies' do
       v2_auth_header
       tags 'Policies'
-      description 'Lists Policies'
+      description 'You can see what policies are currently in place.'\
+                  'Policies are statements that manage API scenarios '\
+                  'such as authentication, rate limiting, caching, requests and responses.'
```
Comment (romanblanco) [inline spec/integration/v2/policies_spec.rb:19]:
```
Use this endpoint to retrieve the list of policies that have been created to test the compliance of your registered systems. 
```

File: spec/integration/v2/systems_spec.rb [hunk 6]
```diff
@@ -299,7 +299,7 @@
 
       v2_auth_header
       tags 'Policies'
-      description 'Assigns a System to a Policy'
+      description 'Use this endpoint to connect a specific system to a policy.'
```
Comment (romanblanco) [inline spec/integration/v2/systems_spec.rb:302]:
```
Use this endpoint to assign a specific system to a specific policy.
```

File: spec/integration/v2/systems_spec.rb [hunk 7]
```diff
@@ -338,7 +338,7 @@
 
       v2_auth_header
       tags 'Policies'
-      description 'Unassigns a System from a Policy'
+      description 'Use this endpoint to remove the connection between a system and a policy.'
```
Comment (romanblanco) [inline spec/integration/v2/systems_spec.rb:341]:
```
Use this endpoint to remove a specific system from a specific policy.
```

File: spec/integration/v2/rules_spec.rb [hunk 8]
```diff
@@ -16,7 +16,7 @@
     get 'Request Rules' do
       v2_auth_header
       tags 'Content'
-      description 'Lists Rules assigned'
+      description 'This is a list of all security guide rules.'
```
Comment (romanblanco) [inline spec/integration/v2/rules_spec.rb:19]:
```
Retrieve a list of rules for a specific security guide.
```

File: spec/integration/v2/rules_spec.rb [hunk 9]
```diff
@@ -69,7 +69,7 @@
     get 'Request a Rule' do
       v2_auth_header
       tags 'Content'
-      description 'Returns a Rule'
+      description 'Retrieve a specific rule.'
```
Comment (romanblanco) [inline spec/integration/v2/rules_spec.rb:72]:
```
Retrieve a specific rule from a specific security guide.
```

File: spec/integration/v2/rules_spec.rb [hunk 10]
```diff
@@ -110,7 +110,7 @@
     get 'Request Rules assigned to a Profile' do
       v2_auth_header
       tags 'Content'
-      description 'Lists Rules assigned to a Profile'
+      description 'This is a list of all security guide rules for a specific profile.'
```
Comment (romanblanco) [inline spec/integration/v2/rules_spec.rb:113]:
```
Retrieve a list of all security guide rules for a specific profile.
```

File: spec/integration/v2/security_guides_spec.rb [hunk 11]
```diff
@@ -13,7 +13,7 @@
     get 'Request Security Guides' do
       v2_auth_header
       tags 'Content'
-      description 'Lists Security Guides'
+      description 'This is a list of all SCAP guides.'
```
Comment (romanblanco) [inline spec/integration/v2/security_guides_spec.rb:16]:
```
Retrieve a list of all SCAP guides.
```

File: spec/integration/v2/supported_profiles_spec.rb [hunk 12]
```diff
@@ -13,7 +13,7 @@
     get 'Request Supported Profiles' do
       v2_auth_header
       tags 'Content'
-      description 'Lists Supported Profiles'
+      description 'List of profiles supported by certain RHEL versions.'
```
Comment (romanblanco) [inline spec/integration/v2/supported_profiles_spec.rb:16]:
```
Retrieve the list of profiles supported by particular RHEL versions.
```

File: spec/integration/v2/value_definitions_spec.rb [hunk 13]
```diff
@@ -15,7 +15,7 @@
     get 'Request Value Definitions' do
       v2_auth_header
       tags 'Content'
-      description 'Lists Value Definitions'
+      description 'List the fields that can be edited within your profile.'
```
Comment (romanblanco) [inline spec/integration/v2/value_definitions_spec.rb:18]:
```
Retrieve a list of the fields which can be edited within a profile.
```

**Review by romanblanco — COMMENTED** [review]
@CodeDuchess Applied other set of suggestions.

**Pull request merged**